src/layouts/SpeakersSection.jsx

```diff
@@ -48,6 +48,9 @@ const SpeakersSection = ({
             className="h-8 md:h-10 lg:h-12"
           />
           <button
+            aria-label={
+              isExpanded ? 'Collapse 2025 Speakers' : 'Expand 2025 Speakers'
+            }
```

Comment:
Same as above

Reply:
Resolved in [d23ee76](https://github.com/GDG-Detroit/devfest-website/pull/40/commits/d23ee76eae8c1c5ea27d0d3694f4b0dc6ee8cbe5)